src/differentials/composite.jl

```diff
@@ -20,13 +20,33 @@ via `comp.fieldname`.
 Any fields not explictly present in the `Tangent` are treated as being set to `ZeroTangent()`.
 To make a `Tangent` have all the fields of the primal the [`canonicalize`](@ref)
 function is provided.
+
+# Examples
+```jldoctest
+julia> Tangent{Symmetric}(; data = [1 2; 3 4])
+Tangent{Symmetric}(data = [1 2; 3 4],)
+
+julia> canonicalize(ans)
+Tangent{Symmetric}(data = [1 2; 3 4], uplo = ZeroTangent())
+
+julia> Tangent(Symmetric([1 2; 3 4]))
+Tangent{Symmetric}(data = [1 2; 3 4], uplo = 'U')
+
+julia> ChainRulesCore.construct(Symmetric, ChainRulesCore.backing(ans))
+ERROR: MethodError: no method matching Symmetric(::Matrix{Int64}, ::Char)
```

Comment:
Tangent is for structs.
Using it on a wrapper array type is pretty weird.
It makes some sense for some concrete array types.
But really it is for structs.
Showing how it works on a AbstractArray in the docs is liable to confuse people.

Better example is to declare a struct  and show how it is used with that.